R/routes.r

```diff
@@ -10,7 +10,7 @@
 #' @param request_path character. The path to which the request was sent.
 #' @return the correct route drawn from the routes list
 determine_route <- function(routes, request_path) {
-  stopifnot(is.list(routes))
+  if(!is.list(routes)) stop("Invalid routes.")
```

Comment:
I'm actually not sure this *is* more informative; maybe "Invalid routes; please provide a list."?